docs/marionette.functions.md

```diff
@@ -124,16 +126,35 @@ var Obj = Mn.Object.extend({
 
   initialize: function(attributes, options){
     this.options = options;
-    var foo = Mn.getOption(this, 'foo');
+    var foo = this.getOption('foo');
     console.log(foo);
-    var foo2 this.getOption('foo');
-    console.log(foo2);
   }
 });
 
-var obj1 = new Obj(); // => "bar"
+var obj1 = new Obj(); // => 'bar'
+var obj2 = new Obj({}, { foo: 'quux' }); // => 'quux'
+
+```
+
+[Live example](https://jsfiddle.net/marionettejs/271nn9ht/)
```

Comment:
this example is forcing the code to work by adding `this.options = options` but that's mostly breaking Marionette.

It is however necessary in a Backbone.Model